packages/app/src/containers/ExportInspector.tsx

```diff
@@ -45,26 +50,117 @@ export default memo(function ExportInspector() {
   const exportFormats = selectedLayer.exportOptions.exportFormats;
 
   // TODO: Handle export formats
+  const handleFileFormat = useCallback(
+    async (exportFormat: Sketch.ExportFormat) => {
+      const {
+        scale,
+        absoluteSize,
+        fileFormat,
+        visibleScaleType,
+      } = exportFormat;
+
+      const adjustSize = {
+        width: Math.ceil(
+          visibleScaleType === Sketch.VisibleScaleType.Scale
+            ? selectedLayer.frame.width * scale
+            : visibleScaleType === Sketch.VisibleScaleType.Width
+            ? absoluteSize
+            : selectedLayer.frame.width,
+        ),
+        height: Math.ceil(
+          visibleScaleType === Sketch.VisibleScaleType.Scale
+            ? selectedLayer.frame.height * scale
+            : visibleScaleType === Sketch.VisibleScaleType.Height
+            ? absoluteSize
+            : selectedLayer.frame.height,
+        ),
+      };
+
+      const data = await renderImageFromCanvas(
+        CanvasKit,
+        adjustSize.width,
+        adjustSize.height,
+        theme,
+        getWorkspaceStateSnapshot(),
+        fileFormat,
+        () => <RCKLayerPreview layer={selectedLayer} size={adjustSize} />,
+      );
+      return data;
+    },
+    [CanvasKit, theme, selectedLayer, getWorkspaceStateSnapshot],
+  );
+
+  const setFileName = useCallback(
+    (exportFormat: Sketch.ExportFormat) => {
+      const {
+        scale,
+        name,
+        namingScheme,
+        fileFormat,
+        visibleScaleType,
+      } = exportFormat;
+
+      return `${!namingScheme && name ? `${name}.` : ''}${selectedLayer.name}${
```

Comment:
We don't need to include the `scale` or `visibleScaleType` in the name. So the logic will be:

- if there's no naming scheme: return `${selectedLayer.name}.${fileFormat}`
- if the naming scheme is Prefix: `${name}${selectedLayer.name}.${fileFormat}`
- if the naming scheme is Suffix: `${selectedLayer.name}${name}.${fileFormat}`